ddtrace/settings/config.py

```diff
@@ -446,6 +446,8 @@ def __init__(self):
         self.version = os.getenv("DD_VERSION", default=self.tags.get("version"))
         self.http_server = self._HTTPServerConfig()
 
+        self._trace_asgi_websocket = os.getenv("DD_TRACE_ASGI_WEBSOCKET", default=False)
```

Comment:
Putting this option on the `asgi` integration config [here](https://github.com/DataDog/dd-trace-py/blob/3b0311d865a03ebb92ac0ba9573166dbeb660f69/ddtrace/contrib/asgi/middleware.py#L31) would avoid leaking integration-specific code into the top-level config object.

Reply:
Ah I hadn't thought about how that's organized, makes sense!

Reply:
I'm not really sure without digging in. You could give it a try and see if you can get the tests to pass. There are a few contrib-specific options in configuration.rst, so you wouldn't be breaking an existing rule by putting this new option at the global level. Even so, I think adding the config to multiple contribs could still be preferable for the sake of encapsulation. Your call.

Reply:
I went with adding it to each config that uses the middleware. Lmk what you think.